R/bundlePackageRenv.R

```diff
@@ -5,8 +5,8 @@ snapshotRenvDependencies <- function(bundleDir,
 
   old <- options(
     renv.verbose = FALSE,
-    renv.consent = TRUE,
-    pkgType = "source")
```

Comment:
Not sure this is correct; need to discuss.

Reply:
Talked this over with @kevinushey, and I think the new behaviour is more correct — this should reduce snapshot time in the worst case where you have an existing lockfile and your library is out of sync.